server/api.js

```diff
@@ -263,4 +266,43 @@ router.post("/bid/:bidId/status", async (req, res) => {
 	}
 });
 
+router.post("/sign-in", async (req, res) => {
+	const { email, password } = req.body;
+
+	if (!email || !password) {
+		return res.status(400).json({});
+	}
+
+	try {
+		const hash = bcrypt.hashSync(password, SALT);
+		const userResult = await db.query(
+			"SELECT * FROM users WHERE email = $1 AND password_hash = $2",
+			[email, hash]
+		);
+
+		const user = userResult.rows[0];
+
+		if (!user) {
+			return res.status(401).json({});
+		}
+
+		const token = uuidv4();
+		const expirationDate = new Date();
+		expirationDate.setMonth(expirationDate.getMonth() + 1);
+
+		if (expirationDate.getDate() !== new Date().getDate()) {
+			expirationDate.setDate(0);
+		}
```

Comment:
What is this?

Reply:
I have changed it back to the way it was before